components/collection/activity/ownerInsightsTabs/HolderTab.vue

```diff
@@ -1,60 +1,73 @@
 <template>
   <div>
-    <div
-      v-for="[holderId, holdings] in displayedHolders"
-      :key="holderId"
-      class="">
-      <div class="is-flex is-flex-direction-column gap">
-        <div class="px-5">
-          <ProfileLink
-            :address="holderId"
-            :avatar-size="35"
-            class="has-text-weight-bold" />
-          <div class="is-flex is-justify-content-space-between mt-2">
-            <span class="is-size-7 has-text-grey">{{
-              $t('activity.owned')
-            }}</span>
-            <span>{{ holdings.nftCount }}</span>
-          </div>
-          <div class="is-flex is-justify-content-space-between">
-            <span class="is-size-7 has-text-grey">{{
-              $t('activity.totalBought')
-            }}</span>
-            <CommonTokenMoney :value="holdings.totalBought" />
-          </div>
-          <div class="is-flex is-justify-content-space-between">
-            <span class="is-size-7 has-text-grey">{{
-              $t('activity.totalSold')
-            }}</span>
-            <CommonTokenMoney
-              v-if="holdings.totalSold > 0"
-              :value="holdings.totalSold" />
-            <span v-else>--</span>
-          </div>
-          <div class="is-flex is-justify-content-space-between">
-            <span class="is-size-7 has-text-grey">{{
-              $t('activity.date')
-            }}</span>
-            <span>{{ timeAgo(holdings.lastActivityTimestamp) }}</span>
-          </div>
-          <div>
-            <div
-              class="is-size-7 has-text-k-blue is-clickable"
-              @click="toggleNFTDetails(holderId)">
-              {{ $t('activity.nftDetails') }}
-              <NeoIcon
-                :icon="
-                  isNFTDetailsOpen[holderId] ? 'chevron-down' : 'chevron-right'
-                " />
+    <template v-if="displayedHolders.length">
```

Comment:
you have :

```
<template v-if=...>
...
<div v-else>
```

for consistency sake i would make both either div or template up to you